tests/integration.rs

```diff
@@ -80,9 +80,9 @@ fn correct_drop() {
     drop(queue.pop().expect("Should be only one popper").expect("Shouldn't be empty"));
     assert_eq!((SIZE + 1, 3), tracker.created_deleted());
 
-    // peek and drop 2
-    drop(queue.peek().expect("Should be only one peeker").expect("Shouldn't be empty"));
-    drop(queue.peek().expect("Should be only one peeker").expect("Shouldn't be empty"));
+    // peek front and drop 2
+    drop(queue.front().expect("Should be only one front peeker").expect("Shouldn't be empty"));
+    drop(queue.front().expect("Should be only one front peeker").expect("Shouldn't be empty"));
```

Comment:
The distinction between front and back peeker is unnecessary here, since they condition on the same atomic bool

Reply:
Swapped the second call to `front()` to `back()`